guide/en/databases/db-migrations.md

```diff
@@ -0,0 +1,98 @@
+# Migrations
+
+To use migrations, install [yiisoft/db-migration](https://github.com/yiisoft/db-migration/) package:
+
+```shell
+composer require yiisoft/db-migration
+```
+
+### Example config with Yii Console
```

Comment:
The context of the guide assumes that we're using Yii console so there's no need to mention it specifically.

Reply:
done